### ISSUE TITLE
Wrap Skeleton, Alert, Pagination, Timeline MUI components

### ISSUE THREAD

**Issue by GewoonMaarten** [nl-design-system/denhaag#7]
[S] = Small, [M] = Medium, [L] = Large, [O] = Optional

A component is finished when:
* documentation is ready in storybook

* all variants are in storybook

* All parameters are documented (briefly, expand after basic styling)

* where applicable, actions linked to components

* versioning added to component (must be below 1.0)

* package defined and added to meta package within Lerna

* review by a colleague after the above tasks are complete before going to Master

* code has been merged to master

* component visible on Storybook production (then subtask on Done)

Applicable after adding Basic Design

* component has been tested functionally

* unit test (discuss with @mjcarsjens)

* accessibility check guide has been completed

**Tasks**

- [ ] [L] Skeleton
- [ ] [M, O] Alert
- [ ] [M] Pagination
- [ ] Timeline

**Comment by ogjtech** [comment]
@Gemeente-DenHaag/mdh-rysst is it worth to keep adding this issue to the backlog? We have been waiting on MUI 5 since November, when we found this out. Unless any of these components have some sort of priority, I would just leave them be and implement them either when fully released and when they are necessary.

**Comment by bddjong** [comment]
@ogjtech that's why we have the blocked label ;). We should still keep track of this issue in my opinion. It's still a desired feature, so it should remain on the backlog.

**Comment by bddjong** [comment]
Issue no longer relevant as we have decided to move away from Material UI
